### DISCUSSION TITLE
Multiple track selection

### DISCUSSION THREAD

**Discussion by Emilot** [jcorporation/myMPD#465] [Ideas]
Hello JC,

Is it possible to have a multiple track selection in Queue card and in the Filesystem View?

In Queue card will be very useful if i can select different tracks and move them up or down or whatever (delete them, etc).

Thank you!

**Comment by jcorporation** [comment]
This could be done and could be useful in all list views.

**Reply by jcorporation** [reply]
I opened an issue #475
